<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>window</class>
 <widget class="QDialog" name="window">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>680</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>680</width>
    <height>300</height>
   </size>
  </property>
  <property name="baseSize">
   <size>
    <width>500</width>
    <height>8000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Texto a voz </string>
  </property>
  <property name="toolTip">
   <string notr="true"/>
  </property>
  <property name="whatsThis">
   <string notr="true"/>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
	border-radius: 30px;
	background-color: rgb(0, 0, 43)

}

QTextEdit {
	border: 2px solid rgb(37,39,48);
	border-radius: 8px;
	color: #FFF;
	padding-left: 10px;
	padding-right: 10px;
	background-color: rgb(34,36,44);
}

QLineEdit {
	border: 2px solid rgb(37,39,48);
	border-radius: 8px;
	color: #FFF;
	padding-left: 10px;
	padding-right: 10px;
	background-color: rgb(34,36,44);
}

QLineEdit:hover {
	border: 2px solid rgb(48,50,62);
}

QLineEdit:focus {
	border: 2px solid rgb(85,170,255);
	background-color: rgb(43,45,56);
}

QLabel {
	color:rgb(132, 132, 132)
}

QPushButton {
	border: 2px solid rgb(37,39,48);
	border-radius: 8px;
	color: #FFF;
	padding-left: 10px;
	padding-right: 10px;
	background-color: rgb(34,36,44);
}</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>280</y>
     <width>681</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QPushButton" name="exit">
      <property name="text">
       <string>Exit program :'(</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>6</y>
     <width>681</width>
     <height>271</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="textEdit">
      <property name="text">
       <string>Waiting for connections...</string>
      </property>
      <property name="textFormat">
       <enum>Qt::PlainText</enum>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignJustify|Qt::AlignTop</set>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
      <property name="openExternalLinks">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>exit</sender>
   <signal>clicked()</signal>
   <receiver>window</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>299</x>
     <y>291</y>
    </hint>
    <hint type="destinationlabel">
     <x>281</x>
     <y>279</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
